glances/rootfs/etc/services.d/influxdb/run

```diff
@@ -5,27 +5,40 @@
 # ==============================================================================
 declare -a options
 
-if bashio::config.false 'influxdb.enabled'; then
+if bashio::config.false 'influxdb.enabled' && bashio::config.false 'influxdb2.enabled'; then
+    # This script is re-run by systemd whenever it terminates, so sleep for a
+    # while to avoid a continual loop.
     exec sleep 86400
 fi
 
-options+=(-C /etc/glances.conf)
-options+=(--export influxdb)
-options+=(--quiet)
+function glances_export() {
+    influxdb_version="$1"
+    sleep_interval="$2"
 
-options+=(--time "$(bashio::config 'refresh_time')")
+    options+=(-C /etc/glances.conf)
+    options+=(--export "$influxdb_version")
```

Comment:
If both options are enabled, the influxdb2 process will export both v1 & v2, while v1 is also active.